sample-config/dev-config/controller.toml

```diff
@@ -10,4 +10,4 @@ domain_name = "ns1.example.com."
 
 [http]
 cluster = "plane.test"
-services.hello-world = "ghcr.io/drifting-in-space/demo-image-hello-world:sha-3b58532"
+services.hello-world = "ghcr.io/drifting-in-space/demo-image-hello-world:latest"
```

Comment:
This is only used for local (manual) testing so I'm fine not pinning it. I updated the image to display `PORT` in order to test this change.